internal/project/framework/internals/helpers/common.go

```diff
@@ -8,9 +8,10 @@ import (
 	"os"
 	"time"
 
-	"{{ .PkgPath }}/framework/dbdrivers"
-	"{{ .PkgPath }}/framework/internals/global"
-	str "{{ .PkgPath }}/framework/internals/string"
+	/**#bean*/
+	"demo/framework/dbdrivers" /*#bean.replace("{{ .PkgPath }}/framework/dbdrivers")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines